### DISCUSSION TITLE
[FEATURE] RSS for Release Notes

### DISCUSSION THREAD

**Discussion by b0g3r** [amzn/selling-partner-api-models#617] [Ideas] [answered]
**Describe the improvement**
RSS channel for [Release Notes](https://developer-docs.amazon.com/sp-api/page/release-notes). It allows setting up notifications about changes in API

**Additional context**
Advertising API already have it: [Release Notes](https://advertising.amazon.com/API/docs/en-us/info/release-notes) and their [RSS](https://d3a0d0y2hgofx6.cloudfront.net/rss/en-us/ad-api-rss.xml)

**Comment by sungolivia** [comment]
This is a great feature request! I am going to move this to Discussions tab to get some more +1s from developers!

**Comment by fegbers** [comment]
In my opinion, it could also be a viable feature for GitHub itself (via RSS or just the GitHub notifications for a particular discussion category)..  
See https://github.com/github/feedback/discussions/31

**Comment by b0g3r** [comment, marked as answer]
Because SP API documentation is based on the readme.io service, it has built-in RSS for changelogs, found it after creating issue: https://developer-docs.amazon.com/sp-api/changelog.rss

I'm not sure that it is actual and informative, but it exists

**Comment by sungolivia** [comment]
I know this is answered but we did put a small link on our main landing page if it helps!
https://developer-docs.amazon.com/sp-api/ 
![Screen Shot 2022-06-15 at 2 46 34 PM](https://user-images.githubusercontent.com/7606170/173935434-abddd75f-a03c-45d6-aed4-82d38d5ff319.png)
